src/lib/cmp.js

```diff
@@ -38,7 +38,13 @@ export default class Cmp {
 				const publisherConsents = (config.storePublisherData && store.getPublisherConsentsObject()) || { lastUpdated: Date.now() }; // if publisher consent is not enabled mark - cookie as valid
 				const shouldBePromted = checkReprompt(config.repromptOptions, vendorConsents, publisherConsents);
 
-				if (config.gdprAppliesGlobally) {
+				if (config.testingMode) {
```

Comment:
We want three cases for a testing mode:
- Set to always show, so the CMP renders every time you load the page no matter what
- Set to never show, so the CMP has to be triggered with Javascript in the console ( `window.__cmp('showConsentTool'` )
- Normal